images/rook/Dockerfile

```diff
@@ -16,5 +16,16 @@ FROM BASEIMAGE
 
 ADD rook rookflex /usr/local/bin/
 
+# Setup user for build execution and application runtime
+ENV ROOK_ROOT=/
+ENV PATH=${ROOK_ROOT}/bin:${PATH} HOME=${ROOK_ROOT}
```

Comment:
I would make the home directory `/home/rook`

Reply:
ok